dev/CommonStyles/Common_themeresources.xaml

```diff
@@ -252,7 +252,7 @@
             <SolidColorBrush x:Key="SystemControlHighlightListAccentMediumLowBrush" Color="{ThemeResource SystemAccentColor}" Opacity="0.75" />
             <x:Boolean x:Key="UseSystemFocusVisuals">True</x:Boolean>
             <Thickness x:Key="TextControlBorderThemeThickness">1</Thickness>
-            <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
+            <Thickness x:Key="TextControlBorderThemeThicknessFocused">1,1,1,2</Thickness>
```

Comment:
This resource is used for a lot of other controls. Is this meant to be a global change?

Reply:
@karenbtlai  The change is also in this PR https://github.com/microsoft/microsoft-ui-xaml/pull/3933/files for all TextBoxes

Reply:
Agreed -- I'm changing this too. :P